doc/classes/EditorInterface.xml

```diff
@@ -48,6 +48,14 @@
 				[b]Note:[/b] This returns the main editor control containing the whole editor, not the 2D or 3D viewports specifically.
 			</description>
 		</method>
+		<method name="get_editor_scale" qualifiers="const">
+			<return type="float">
+			</return>
+			<description>
+				Returns the actual scale of the editor UI. Can be used to adjust position and dimensions of the UI added by plugins.
+				[b]Note:[/b] Value is set via the editor settings.
```

Comment:
```suggestion
				[b]Note:[/b] This value is set via the [code]interface/editor/display_scale[/code] editor setting.
```

Reply:
I was actually referring to the UI, because it requires a restart anyway. I don't want to confuse people into setting it via code like that and seeing no immediate effect, or worse, glitches due to mismatched old and new values.